Purchases/FoundationExtensions/NSDictionary+RCExtensions.m

```diff
@@ -0,0 +1,34 @@
+//
+//  NSDictionary+RCExtensions.m
+//  Purchases
+//
+//  Created by Andrés Boedo on 9/29/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+#import "NSDictionary+RCExtensions.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@implementation NSDictionary (RCExtensions)
+
+- (NSDictionary *)removingNSNullValues {
```

Comment:
turns out this wasn't necessary, but it might be a useful utility. I think there's actually a couple of places where we're doing this manually. So I wasn't sure whether to remove it - there are also tests for it, so I imagine it can't hurt to have it around.